examples/plotting/time_distance.py

```diff
@@ -148,12 +149,19 @@
 imag_ax = reprojected_sub_maps[0].plot(axes=left_ax, clip_interval=[1, 99]*u.percent)
 left_ax.plot_coord(line_coords)
 
-right_ax['repro'].imshow(intensities_reproject.T, aspect='auto', interpolation='none',
-                         extent=extent, cmap=imag_ax.get_cmap())
-right_ax['trans'].imshow(intensities_transform.T, aspect='auto', interpolation='none',
-                         extent=extent, cmap=imag_ax.get_cmap())
-right_ax['diff'].imshow((intensities_reproject-intensities_transform).T, interpolation='none',
-                        aspect='auto', extent=extent, cmap='bwr')
+right_ax['repro'].imshow(
+    intensities_reproject.T, aspect='auto', interpolation='none',
+    extent=extent, cmap=imag_ax.get_cmap()
+)
+right_ax['trans'].imshow(
+    intensities_transform.T, aspect='auto', interpolation='none',
+    extent=extent, cmap=imag_ax.get_cmap()
+)
+right_ax['diff'].imshow(
+    (intensities_reproject-intensities_transform).T, interpolation='none',
+    aspect='auto', extent=extent, cmap='bwr'
+)
+plt.colorbar(right_ax['diff'].images[0], ax=right_ax['diff'], label='Difference')
```

Comment:
I think maybe the main image also needs a colour bar, otherwise you don't have a reference for the scale on the difference image. Like is the min/max of the main image 0<>10 if so the +-5 on the difference is a lot etc.